browser/request_otr/sources.gni

```diff
@@ -3,25 +3,18 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at https://mozilla.org/MPL/2.0/.
 
-import("//brave/components/request_otr/common/buildflags/buildflags.gni")
+brave_browser_request_otr_sources = [
+  "//brave/browser/request_otr/request_otr_service_factory.cc",
+  "//brave/browser/request_otr/request_otr_service_factory.h",
+  "//brave/browser/request_otr/request_otr_tab_helper.cc",
+  "//brave/browser/request_otr/request_otr_tab_helper.h",
+]
 
-brave_browser_request_otr_sources = []
-brave_browser_request_otr_deps = []
-
-if (enable_request_otr) {
-  brave_browser_request_otr_sources += [
-    "//brave/browser/request_otr/request_otr_service_factory.cc",
-    "//brave/browser/request_otr/request_otr_service_factory.h",
-    "//brave/browser/request_otr/request_otr_tab_helper.cc",
-    "//brave/browser/request_otr/request_otr_tab_helper.h",
-  ]
-
-  brave_browser_request_otr_deps += [
-    "//base",
-    "//brave/components/request_otr/browser",
-    "//brave/components/request_otr/common",
-    "//brave/extensions:common",
-    "//chrome/common",
-    "//components/keyed_service/content",
-  ]
-}
+brave_browser_request_otr_deps = [
+  "//base",
+  "//brave/components/request_otr/browser",
+  "//brave/components/request_otr/common",
+  "//brave/extensions:common",
```

Comment:
is this needed? I'm not seeing includes for it

Reply:
removed in https://github.com/brave/brave-core/commit/c83ae23838b8132c40d9ce007c338abf52804ae9